geopandas_view/view.py

```diff
@@ -66,8 +66,9 @@ def view(
     column : str, np.array, pd.Series (default None)
         The name of the dataframe column, np.array, or pd.Series to be plotted.
         If np.array or pd.Series are used then it must have same length as dataframe.
-    cmap : str (default None)
-        The name of a colormap recognized by matplotlib or a list-like of colors.
+    cmap : str, branca.colormap, self-defined function (default None)
+        The name of a colormap recognized by matplotlib, a list-like of colors,
+        branca.colormap object or a self-defined function
```

Comment:
Can you document the required signature and return of the self-defined function?